src/nvim/mouse.c

```diff
@@ -308,10 +308,11 @@ int jump_to_mouse(int flags,
   if (ui_has(kUIMultigrid)) {
     gp = &curwin->w_grid;
```

Comment:
one worry I have is that we dont use   `int grid = mouse_grid;` ? we are not necessarily clicking on the current window

Reply:
@teto 
That's true.
Then I will think about using `wp` instead of `curwin`.

I'd like to know your opinion on two related points.

1.
I think it is correct to use `default_grid` when `ext_multigrid` is invalid.
What do you think?
2.
`curwin` is used not only for retrieving clicked characters but also for comparing retrieved characters, but I think it should be changed to use `wp` as well.